### PR TITLE
[4.0] Change 'Module Class Suffix' to 'Module Class'

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #16894  .

### Summary of Changes
Inline with #17026 this PR changes 'Module Class Suffix' to 'Module Class'

### Testing Instructions
Add class to module using the 'Module Class' field (no space). Check frontend and ensure class has been added correctly.

Eg.
`<div class="card yourClass">`

### Documentation Changes Required
Yes

### CODE REVIEW

File: administrator/language/en-GB/en-GB.com_modules.ini [hunk 1]
```diff
@@ -36,7 +36,7 @@ COM_MODULES_FIELD_CACHING_LABEL="Caching"
 COM_MODULES_FIELD_CLIENT_ID_LABEL="Module Location"
 COM_MODULES_FIELD_CONTENT_LABEL="Text"
 COM_MODULES_FIELD_MODULE_LABEL="Module Type"
-COM_MODULES_FIELD_MODULECLASS_SFX_LABEL="Module Class Suffix"
```
Comment (wilsonge) [inline administrator/language/en-GB/en-GB.com_modules.ini:39]: Can we leave this string constant as is please? Huge numbers of modules use this. I know it kinda sucks as a descriptive thing - but it's also going to cause stupid pain

Comment (ciar4n) [reply administrator/language/en-GB/en-GB.com_modules.ini:39]: Makes sense.. done

File: templates/system/html/modules.php [hunk 2]
```diff
@@ -95,7 +95,7 @@ function modChrome_xhtml($module, &$params, &$attribs)
 
 	// Temporarily store header class in variable
 	$headerClass    = $params->get('header_class');
-	$headerClass    = $headerClass ? ' class="' . htmlspecialchars($headerClass, ENT_COMPAT, 'UTF-8') . '"' : '';
+	$headerClass    = $headerClass ? ' class=" ' . htmlspecialchars($headerClass, ENT_COMPAT, 'UTF-8') . '"' : '';
```
Comment (Quy) [inline templates/system/html/modules.php]: No need to add a space here.

File: templates/system/html/modules.php [hunk 3]
```diff
@@ -95,7 +95,7 @@ function modChrome_xhtml($module, &$params, &$attribs)
...
 
 	if (!empty ($module->content)) : ?>
 		<<?php echo $moduleTag; ?> class="moduletable<?php echo htmlspecialchars($params->get('moduleclass_sfx'), ENT_COMPAT, 'UTF-8') . $moduleClass; ?>">
```
Comment (Quy) [inline templates/system/html/modules.php]: Add a space here?

Comment (ciar4n) [reply templates/system/html/modules.php]: Thank you @Quy

**Pull request merged**